fortio_main.go

```diff
@@ -391,6 +396,14 @@ func fortioLoad(justCurl bool, percList []float64) {
 		RunID:       *bincommon.RunIDFlag,
 		Offset:      *offsetFlag,
 	}
+	if *accessLogFileFlag != "" {
+		al, err := periodic.NewFileAccessLogger(*accessLogFileFlag, *accessLogFileFormat)
+		if err != nil {
+			_, _ = fmt.Fprintf(out, "Aborting because of %v\n", err)
```

Comment:
this would typically be
```
			log.Fatalf("Unable to create access logger: %v", err)
```
(even if the stack trace can be confusing to the users, if we expect this to happen a lot then
```
log.Errf("Aborting because of access logger error: %v", err)
os.Exit(1)
```

Reply:
done, also added a more direct method on the runner options